src/keri/core/eventing.py

```diff
@@ -5806,7 +5806,7 @@ def processEscrowDelegables(self):
                     self.processEvent(serder=eserder, sigers=sigers, wigers=wigers, delseqner=seqner,
                                       delsaider=saider, local=esr.local)
                 else:
-                    raise MissingDelegableApprovalError()
+                    raise MissingDelegableApprovalError("No delegation seal found for event.")
```

Comment:
might be good to include the event digest?



```MissingDelegableApprovalError("Missing Delegable Approval for evt at dig = {}.""".format(bytes(edig)))```

Reply:
Yes, that would be better